package.json

```diff
@@ -48,6 +40,17 @@
     "lint:fix": "eslint --fix --ext .js,.jsx,.ts,.tsx ./src",
     "format": "prettier --write --ignore-unknown src/"
   },
+  "lint-staged": {
+    "src/**/*.{js,jsx,ts,tsx ./src}": [
+      "prettier --write",
+      "eslint"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
```

Comment:
Looks like a legacy husky config? is this working? I'm expecting a .husky folder and a prepare script